content/docs/compliance/domain-standards.md

```diff
@@ -34,3 +34,11 @@ If you do need DNSSEC for your custom domain, you are responsible for configurin
 cloud.gov supports mapping your DNSSEC-enabled custom domain to your applications hosted on cloud.gov -- see [DNSSEC support for the CDN service]({{< relref "docs/services/cdn-route.md#dnssec-support" >}}) and [DNSSEC support for the custom domain service]({{< relref "docs/services/custom-domains.md#dnssec-support" >}}).
 
 *Additional details are available the cloud.gov System Security Plan, including controls SC-20, SC-21, SC-22, and SC-23.*
+
+## Certificates
+
+cloud.gov's global TLS certificate is issued by a 3rd party, open source vendor known as [Let's Encrypt](https://letsencrypt.org/). Let's Encrypt allows for individuals, companies, or agencies to request free TLS certificates in an automated fashion for easy HTTPS functionality. As part of that, the cloud.gov platform does not control any aspect of the certificate provisioning process other than the common name and alternative name. We do not control the ciphers, certificate validity, or other algorithms used in modern TLS certificate creation.
```

Comment:
this is only partially correct. The cipher suite issues are caused by us leveraging A/ELBs and CloudFront, not anything to do with LE

Reply:
I'll add lingo to reference the ELB stuff.